hack/collector-default

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+HOST_PATH=$1
+BUNDLE_DIR=$2
+
+# RKE2: Logs for the kubelet are save in rke2 agent log folder
+RKE2_KUBELET_LOG_PATH=${HOST_PATH}/var/lib/rancher/rke2/agent/logs/kubelet.log
+
+function collect_sysinfo(){
+    cd ${BUNDLE_DIR}
+    mkdir -p hostinfos
+    cd hostinfos
+    
+    echo "hostname : " `cat ${HOST_PATH}/etc/hostname` > hostinfo
+    echo "os-release : " `cat ${HOST_PATH}/etc/os-release` >> hostinfo
+    echo "system-info : " `uname -a` >> hostinfo
+
+    # Boot Configuration
+    cat ${HOST_PATH}/boot/config-$(uname -r) > kernel_config
+}
+
+function collect_dmesg(){
+    dmesg -HTx &> dmesg.log
+}
+
+function collect_syslog(){
+    tail -c 10m ${HOST_PATH}/var/log/syslog > syslog.log
```

Comment:
Just curious, would we need to consider the rotated syslog?
(e.g. syslog, syslog.1 syslog.2.gz ...etc)

Reply:
I will fix it to pack the files starting with syslog under the /var/log folder together.